### PR TITLE
Use tat from application

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds application turn around time in the cg status cases command output.

**How to test**:
1. install on stage of hasta
1. activate stage: `us`
1. run following command: `cg status cases`

**Expected outcome**:
`cg` should show the maximum tat found on the samples for each case, the max-tat is shown rightmost in the output.
If samples are undelivered and tat and max-tat are equal then the rowcolor should be yellow,
if less then green or samples are delivered and tat and max-tat are equal, if grater then red
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [ ] tests executed by @patrikgrenfeldt 
- [ ] "Merge and deploy" approved by @ingkebil 
Thanks for filling in who performed the code review and the test!

This is |minor| **version bump** because no breaking changes

### CODE REVIEW

File: cg/store/api/status.py [hunk 1]
```diff
@@ -675,3 +678,11 @@ def _calculate_date_delta(default, first_date, last_date):
         if first_date:
             delta = (last_date - first_date).days
         return delta
+
+    @staticmethod
+    def _get_max_tat(links):
+        max_tat = 0
+        for link in links:
+            if link.sample.application_version.application.turnaround_time:
+                max_tat = max(0, link.sample.application_version.application.turnaround_time)
```
Comment (ingkebil) [inline cg/store/api/status.py]: nice one

**Comment by ingkebil** [comment]
Really sweet!

**Comment by patrikgrenfeldt** [comment]
before:
image
after:
image
👍

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**